.gitlab-ci.yml

```diff
@@ -0,0 +1,75 @@
+stages:
+  - build
+
+.build:
+  stage: build
+
+  script:
+    - cmake --build _build --config Release -j8
+    - cd _build
+    - cpack -G ZIP
+
+  artifacts:
+    paths:
+      - _build/antaresXpansion-*.*.*-*.zip
+
+ubuntu/x64:
+   image: ubuntu
+   extends: .build
+   before_script:
+     - export DEBIAN_FRONTEND=noninteractive
```

Comment:
i think that in gitlab CI all command are launch as root. This .yml was not fully tested on gitlab CI.
We keep it in case we create a mirror repository in the future